virttest/qemu_vm.py

```diff
@@ -3945,7 +3945,7 @@ def create(
                     ", ".join([str(tid) for tid in self.vcpu_threads]),
                 )
             vhost_thread_pattern = params.get(
-                "vhost_thread_pattern", r"\w+\s+(\d+)\s.*\[vhost-%s\]"
+                "vhost_thread_pattern", r"\w+\s+(\d+)\s.*\[*vhost-%s\]*"
```

Comment:
Hi, @quanwenli 
Could you provide the example output of both situations?
Thanks.
CC @leidwang @PaulYuuu , it maybe affect your test case.

Reply:
Hi @YongxueHong I guess you'd like to ping me istead of @leidwang. And the code looks good to me.

Reply:
on old system:  
 ps aux | grep host
 root      331500 24.8  0.0      0     0 ?        R    22:29   0:18 [vhost-331496]
ps -eL | grep host
 331500  331500 ?        00:00:27 vhost-331496

Reply:
The matched group is not the same one. The old pattern matches "PID" from `ps aux`, but the new pattern matches "LWP" from `ps -eL`, In your sanity test, they are the same, but cannot mark sure they are always the same.

Header of `ps -eL` and `ps aux`
```bash
# ps -eL
    PID     LWP TTY          TIME CMD
      1       1 pts/0    00:00:00 bash
     37      37 pts/0    00:00:00 ps
# ps aux
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  0.0  0.0   4832  3656 pts/0    Ss   02:44   0:00 /bin/bash
root          38  0.0  0.0   7708  3704 pts/0    R+   02:48   0:00 ps aux
```

I suggest changing the pattern accordingly.